pyproject.toml

```diff
@@ -10,7 +10,8 @@ name = "sot"
 authors = [{ name = "Kumar Anirudha", email = "dev@anirudha.dev" }]
 description = "System Oberving Tool"
 readme = "README.md"
-license = { file = "LICENSE" }
+license = "MIT"
+license-files = ["LICENSE"]
```

Comment:
While we're at it, I switched to using the SPDX license expression and the `license-files` field (see https://peps.python.org/pep-0639/).